__tests__/app.test.js

```diff
@@ -93,6 +94,41 @@ describe("GET /api/articles/:article_id", () => {
         expect(body).toEqual({ msg: "Bad request: invalid format" });
       });
   });
+
+  test("404: if article_id is a number, but it does not exist in the DB", () => {
+    return request(app)
+      .get("/api/articles/9999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "Article 9999 not found" });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: This will get all comments from an specific article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+        expect(body.length).toBeGreaterThan(0);
+        body.forEach((comment) => {
+          expect(Object.keys(comment)).toHaveLength(6);
+          expect(comment).toMatchObject({
+            article_id: 1,
```

Comment:
Nice! Well done for making sure the test is still flexible but that we definitely have received back the comments related to the correct article regardless of what the comments info is!